plugins/bookshortcuts.koplugin/main.lua

```diff
@@ -54,25 +72,44 @@ function BookShortcuts:addToMainMenu(menu_items)
 end
 
 function BookShortcuts:getSubMenuItems()
+    local FM_text = _("Open FM")
+    local last_text = _("Last Book")
```

Comment:
`Last book` , and `Open FM` should probably be less familiar :) `Open file browser` or `Open file manager` (dunno what official name for it we may already use elsewhere).

Reply:
Browser.

Reply:
And uncapitalized in `Start with: file browser`.